README.md

```diff
@@ -123,27 +176,49 @@ get_built_package() {
 # export DATE_NUM="$(date -u +%Y%m%d%H%M%S)"
 # export DATE_STR="$(date -u +%Y%m%d_%H%M%S)"
 #
-# Or lock to date/time of the last commit on git, as Travis CI config does
-# (see .travis/common.sh)
+# Or lock to date/time of the last commit on git, as it's done in the CI:
 #
-export DATE_TS="$(git log --format=%ct -n1)"
+DATE_TS="$(git log --format=%ct -n1)"
 export DATE_NUM="$(date --date=@${DATE_TS} -u +%Y%m%d%H%M%S)"
 export DATE_STR="$(date --date=@${DATE_TS} -u +%Y%m%d_%H%M%S)"
+```
+
+### Preparing and building the package
+
+After getting all prerequisites and setting the required variables, it
+is recommended to prepare the recipe before building, as it gives you
+the advantages described on [the `conda-build-prepare`'s
+GitHub page](https://github.com/litex-hub/conda-build-prepare).
+The tool is also used within the CI, which makes the locally built
+package much more similar to the one built by the CI workflow.
+
+The `WORKDIR` variable is the name of the directory that will be
+created  as an output of running `conda-build-prepare`.
+It will contain these subdirectories:
+* `conda-env`,
+* `git-repos` (if the recipe contains any git sources),
+* `recipe`.
+`WORKDIR` can thus be any valid directory name.
 
-# Combinations taken from .travis.yml
-TOOLCHAIN_ARCH=lm32
-export PACKAGE TOOLCHAIN_ARCH
+The arguments passed with `--channels` and `--packages` switches to the
+`conda-build-prepare` are similar to those used within the CI workflow.
 
-cd conda-hdmi2usb-packages
+```bash
+# Prepare the RECIPE with `conda-build-prepare`
+ADDITIONAL_PACKAGES="conda-build=3.20.3 conda-verify jinja2 pexpect python=3.7"
```

Comment:
I'm confused about why `ADDITIONAL_PACKAGES` is needed?

Reply:
This is what we use in the CI so if we seek the maximum similarity with the CI then locally these should also be used.
I'm quite sure that only these set with versions make any difference so the rest (`conda-verify jinja2 pexpect`) can be dumped both from the CI and from here. Should I do it?

Reply:
In the end I've removed `conda-verify jinja2 pexpect` from here as I'm pretty sure they have zero influence on building.

`conda-build` and `python` have been left with versions for maximal CI resemblence.